modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/rest/api/server/organization/management/v1/OrganizationManagementFailureTest.java

```diff
@@ -372,4 +378,30 @@ public void deleteOrganizations() {
                     .statusCode(HttpStatus.SC_NO_CONTENT);
         }
     }
+
+    @Test(dependsOnMethods = "testUpdateDiscoveryAttributesUnauthorized")
+    public void testGetPaginatedOrganizationsWithInvalidLimit() {
+
+        String invalidLimitUrl = ORGANIZATION_MANAGEMENT_API_BASE_PATH + QUESTION_MARK + LIMIT_QUERY_PARAM + "-1";
```

Comment:
Shall we add this string in one constant? like https://github.com/wso2/product-is/blob/684258f1148a1bbdcb0741761c3e08957e396a7e/modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/rest/api/user/application/v1/UserDiscoverableApplicationSuccessTest.java#L44

Then we can just add that like the below.
`Response response = getResponseOfGetWithOAuth2(INVALID_LIMIT_URI_CONST, m2mToken);`